### ISSUE TITLE
Smoothing shifts mesh coordinates

### ISSUE THREAD

**Issue by NikDrummond** [mikedh/trimesh#1070]
I've noticed that mesh smoothing seems to shift the mesh in space:

```
mesh = trimesh.load_mesh('../mesh.stl')
mesh2 = mesh.copy()
mesh2 = trimesh.smoothing.filter_laplacian(mesh2)


scene = trimesh.Scene([mesh,mesh2])
scene.show()
```

Where I would think these meshes should be overlapping. The results are the same for `trimesh.smoothing.filter_taubin()` and `trimesh.smoothing.filter_humphrey()`

![Screenshot from 2020-11-30 16-11-36](https://user-images.githubusercontent.com/42806335/100627002-c1e95400-3326-11eb-898e-8757e857e781.png)

**Comment by mikedh** [comment]
Yeah just looking quickly there's a number of operations that look like they'd produce offsets, like [here](https://github.com/mikedh/trimesh/blob/master/trimesh/smoothing.py#L76-L81) where it's just multiplying by a scaling factor. I think that logic should probably be something like `((vertices - center_mass) * scale) + center_mass`

**Comment by postnubilaphoebus** [comment]
Hi, thanks again for this awesome library with so many functionalities, looks like a lot of work!
Is there an update on the shifting of coordinates in space?
The problem is still happening.
